docs/8-existing-tools.qmd

```diff
@@ -16,119 +16,331 @@ author:
 
-## Include section headers as appropriate
+In this chapter, we answer the questions of how do you find new-to-you AI models and how do you assess whether a new-to-you model or tool will meet your needs?
+In this chapter, we first help you assess your needs.
+We then introduce a way of categorizing tools and help you use your needs assessment to select the right tool category.
+We suggest several places where you can find AI models for bioimage analysis and describe how to assess how well tools in those locations meet your needs.
+Finally, we show a couple of case studies that fulfill different requirements.
 
-Use markdown heading level two for section headers. You can use standard markdown formatting, for example _emphasize the end of this sentence_.
+## Assessing your requirements for a tool (in a perfect world)
 
-This is a new paragraph with more text. Your paragraphs can cross reference other items, such as @fig-simple. Use `fig` to reference figures, and `eq` to reference equations, such as @eq-stddev.
+Before finding and selecting an AI tool, you first need to decide on the type of tool you need.
+Your tool selection will be influenced by a number of factors and you should first honestly answer the questions that follow.
 
-###  Sub-subsection headers are also available
+::: {.callout-note icon=false id=Figure1}
+## Figure 1: Workflows contain modular components
+![Workflows contain modular components](8-existing-tools_files/workflows.png "Workflows contain modular components")
+**Figure 1.**
+When thinking of your workflow and how you might incorporate an AI tool, consider that a workflow (left of image) typically consists of multiple, modular subcomponents (right of image).
+You may be able to replace your whole workflow with an appropriate AI tool or you may be able to replace only a subcomponent while still keeping many of the original workflow subcomponents the same.
+Replacing a single subcomponent with an AI tool may force you to change surrounding subcomponents in your workflow - either with new tools or new configurations of existing tools - as the AI tool may require different inputs and produce different outputs.
 
-To make your sections cross reference-able throughout the book, include a section reference, as shown in the header for @sec-equation.
+Though you may have built a workflow across multiple softwares, it is worth considering whether a single software (or a reduced number of softwares) can run your workflow and part of this consideration includes assessing how well your desired AI tool can run from within other software.
+Some examples of softwares that can be used to run other softwares include CellProfiler @Stirling2021, Fiji @Schindelin2012, Bilayers @Shah, ZeroCostDL4Mic @von_chamier2021 or DL4MicEverywhere @Hidalgo-Cenalmor2024, and Jupyter notebooks @jupyter.
```

Comment:
@ErinWeisbart can you add the von_chamier to the bibliography?

Reply:
It's one of the 3 references that were in the [initial example references.bib in the repo](https://github.com/aicjanelia/BioImagingAI/blob/60d6c8db2498cdfea2b3b4a59f58d5cd632946f6/docs/references.bib#L19) and so is also present in [our version of the references.bib](https://github.com/aicjanelia/BioImagingAI/blob/54887d96ce81e4e23f6bb623c5784625058364dd/docs/references.bib#L19). Perhaps there are formatting mistakes that are preventing it from being auto-recognized?

Reply:
Ah ok, it wasn't searchable for me, all good, ty!